python/cuml/cuml_accel_tests/upstream/scikit-learn/xfail-list.yaml

```diff
@@ -1045,6 +1045,12 @@
   strict: false
   tests:
   - "sklearn.utils.tests.test_estimator_html_repr::test_show_arrow_pipeline"
+- reason: Test is flaky with cuml.accel
+  marker: cuml_accel_flaky
+  condition: scikit-learn>=1.7
+  strict: false
+  tests:
+  - "sklearn.cluster.tests.test_k_means::test_kmeans_elkan_iter_attribute"
```

Comment:
This seems unrelated.

Reply:
It probably is, but it still is a flaky test.

Reply:
When you run this locally? We have not observed it in CI, I believe.

Reply:
Yes, I regularly catch it locally.